migrations/1693569227_addAuthors.ts

```diff
@@ -0,0 +1,60 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newItemTypes: Record<string, SimpleSchemaTypes.ItemType> = {};
+
+  console.log("Create new models/block models");
+
+  console.log('Create block model "Section Cases List" (`section_cases_list`)');
+  newItemTypes["2435781"] = await client.itemTypes.create(
+    {
+      name: "Section Cases List",
+      api_key: "section_cases_list",
```

Comment:
Sorry I did not mention this before but I want to keep new block models prefixed with `section_` reserved for the generic [...slug] page. It's the only distinction we have between 'old' and 'new/generic' models, could you remove it?
Once we cross the bridge to add this section to the generic page we can think of renaming it.